src/store/getters.js

```diff
@@ -72,11 +72,11 @@ export default {
     return newClovers.length
   },
 
-  savedClovers ({ account, allSavedClovers }) {
+  picks ({ account, allSavedClovers }) {
     return allSavedClovers[account || 'anon'] || []
   },
-  cloversFound (_, { savedClovers }) {
```

Comment:
cloversFound is still called inside of AppHeader, is it supposed to be changed to pickCount?